mix.exs

```diff
@@ -18,9 +21,17 @@ defmodule SSHEx.Mixfile do
 
   defp package do
     [
-      maintainers: ["Rubén Caro"],
+      maintainers: ["Niklas Johansson"],
```

Comment:
I am a little unsure what we should put here :thinking: . Should we have more names or only one.